### PR TITLE
Update Danish Translation (88% / 312)

### PR DESCRIPTION
**Pull request by Stisen1**
- Focused translation of campaign scenario descriptions and everything related to this.
- Some changes to names and titles that could be conflicting with campaign related translations
- Some changes to older strings, where the wordings sounds wierd in context of playing the game.

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -3647,21 +3808,26 @@ msgid "Hard"
 msgstr "Svær"
 
 msgid "Campaign Difficulty"
-msgstr ""
+msgstr "Felttogs niveau"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:3811]: Yup

File: files/lang/dk.po [hunk 2]
```diff
@@ -3647,21 +3808,26 @@ msgid "Hard"
...
 
 msgid ""
 "Choose this difficulty if you want to prefer game story over challenge. AI "
 "is weaker in comparison with normal difficulty."
 msgstr ""
+"Vælg dette niveau, hvis du vil foretrække spilhistorie frem for udfordring. "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:3817]: It is allowed in danish, but to say "hvis du foretrækker" means the same thing... And it sounds better. So it has been changed.

File: files/lang/dk.po [hunk 3]
```diff
@@ -1114,244 +1161,335 @@ msgid ""
...
 msgid "The Shrouded Isles"
-msgstr ""
+msgstr "Tågernes Ø"
 
 msgid ""
 "Your mission is to vanquish the warring mages in the magical Shrouded Isles. "
 "The completion of this task will give you a fighting chance against your "
 "rivals."
 msgstr ""
+"Din mission er at overvinde de genstridige troldmænd på Tågeøerne. "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:1402]: Now identical with the scenario-titel

File: files/lang/dk.po [hunk 4]
```diff
@@ -1114,244 +1161,335 @@ msgid ""
...
-msgstr ""
+msgstr "Magtens Ende"
 
 msgid "The Eternal Scrolls"
-msgstr ""
+msgstr "De Evige Skriftruller"
 
 msgid "The Shrouded Isles"
-msgstr ""
+msgstr "Tågernes Ø"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:1396]: Right

File: files/lang/dk.po [hunk 5]
```diff
@@ -921,77 +921,84 @@ msgid "Dwarven Alliance"
...
 msgid "Force of Arms"
-msgstr ""
+msgstr "Våbenstyrke"
 
 msgid "Save the Dwarves"
 msgstr "Red dværgene"
 
 msgid "Carator Mines"
-msgstr ""
+msgstr "Karator Miner"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:960]: Now it is

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , thank you so much for the update!

**Pull request merged**